### PR TITLE
16.04 ESM page discusses 'april 2021' in the future tense

### LINKED ISSUE #10067: 16.04 ESM page discusses 'april 2021' in the future tense
Hello, the 16.04 ESM page discusses the start of 16.04 ESM support as if it is in the future:

- "Xenial enters the ESM period in April 2021"

- "ESM for Ubuntu 16.04 LTS will be available April 2021" twice

Thanks

---
*Reported from: https://ubuntu.com/16-04*

### PR DESCRIPTION
**Pull request by ClementChaumel**
## Done

- Updated the copy according to the [copydoc](https://docs.google.com/document/d/1wclofqY9b3v_M9igpkYn6or5uXx0B_Xb_nVVzGBbOLY/edit#)

## QA

- go on https://ubuntu-com-10164.demos.haus/16-04
- check that the copy matches the copydoc

## Issue / Card

Fixes #10067

### CODE REVIEW

File: templates/16-04/index.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
       <h1>Ubuntu 16.04 LTS <br class="u-hide--small">(Xenial Xerus)</h1>
       <p>Ubuntu 16.04 LTS (Xenial Xerus) was released on 21 April 2016 and introduced us to snaps, a more secure app format, with faster and simpler updates, as well as LXD, the pure-container hypervisor and OpenStack Mitaka. </p>
       <h2 class="p-heading--4">Is Ubuntu 16.04 LTS still supported?</h2>
-      <p class="u-sv2">Yes, Ubuntu 16.04 LTS is supported until 2024 through Canonical’s Extended Security Maintenance (ESM) product. Xenial enters the ESM period in April 2021, with security patches provided for an additional three years beyond the traditional five-year standard support.</p>
+      <p class="u-sv2">Yes, Ubuntu 16.04 LTS is supported until 2024 through Canonical’s Extended Security Maintenance (ESM) product. Xenial entered the ESM period in April 2021, with security patches provided for an additional three years beyond the traditional five-year standard support.</p>
```
Comment (petermakowski) [inline templates/16-04/index.html:14]:
```suggestion
      <p class="u-sv2">Yes, Ubuntu 16.04 LTS is supported until 2026 through Canonical’s Extended Security Maintenance (ESM) product. Xenial entered the ESM period in April 2021, with security patches provided for an additional three years beyond the traditional five-year standard support.</p>
```

File: templates/16-04/index.html [hunk 2]
```diff
@@ -211,10 +211,10 @@ <h4>
         Infrastructure packages
       </h4>
       <p>
-        Canonical will provide extended security maintenance to all binary packages that reside in Ubuntu Main Repository.
+        Canonical provides extended security maintenance to all binary packages that reside in Ubuntu Main Repository.
```
Comment (petermakowski) [inline templates/16-04/index.html:214]: https://docs.google.com/document/d/1wclofqY9b3v_M9igpkYn6or5uXx0B_Xb_nVVzGBbOLY/edit#heading=h.tbcblwt3kct9

```suggestion
        For Ubuntu 16.04 LTS, where technically feasible, Canonical provides extended security maintenance to all binary packages that reside in the Ubuntu Main Repository.
```

File: templates/16-04/index.html [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {% extends "16-04/base_16-04.html" %}
 
 {% block title %}Ubuntu 16.04 LTS (Xenial Xerus) {% endblock %}
-{% block meta_description %}Ubuntu 16.04 LTS (Xenial Xerus) enters the Extended Security Maintenance (ESM) period on 30 April 2021. ESM provides Ubuntu 16.04 LTS continued support and coverage beyond EOL with security patches for an additional three years.{% endblock %}
+{% block meta_description %}Ubuntu 16.04 LTS (Xenial Xerus) entered the Extended Security Maintenance (ESM) period on 30 April 2021. ESM provides Ubuntu 16.04 LTS continued support and coverage beyond EOL with security patches for an additional three years.{% endblock %}
```
Comment (petermakowski) [inline templates/16-04/index.html:4]:
```suggestion
{% block meta_description %}Ubuntu 16.04 LTS (Xenial Xerus) entered the Extended Security Maintenance (ESM) period on 30 April 2021. ESM provides Ubuntu 16.04 LTS continued support and coverage beyond EOL with security patches for an additional five years.{% endblock %}
```

**Review by petermakowski — COMMENTED** [review]
Unless I misunderstand how you work with the copydocs (please correct me if I'm wrong), there are a few things that need updating. See Suggested changes below.

File: templates/16-04/index.html (hunk 1 above)
Comment (ClementChaumel) [reply templates/16-04/index.html:14]: If I understand the comment correctly we need to wait until September 15th to make that change 
image

File: templates/16-04/index.html (hunk 3 above)
Comment (ClementChaumel) [reply templates/16-04/index.html:4]: Same thing here 
image

**Comment by ClementChaumel** [comment]
Good catch on the Infrastructure package paragraph, for the other ones I think we should wait

File: templates/16-04/index.html (hunk 3 above)
Comment (petermakowski) [reply templates/16-04/index.html:4]: Got it! Didn't have the comments in view. Thanks for explaining 🙌

**Review by petermakowski — APPROVED** [review]

**Pull request merged**